### PR TITLE
Fixed #24988 -- Documented passing a dictionary of ValidationErrors to ValidationError

### PR DESCRIPTION
**Pull request by adambrenecki**
Patch for https://code.djangoproject.com/ticket/24988

### CODE REVIEW

File: docs/ref/forms/validation.txt [hunk 1]
```diff
@@ -205,6 +205,15 @@ with ``code``\s and ``params`` but a list of strings will also work::
         _('Error 2'),
     ])
 
+If you detect multiple errors in multiple fields in your form, for instance
```
Comment (loic) [inline docs/ref/forms/validation.txt]: This will **only** work in `Form.clean()`, so "for instance" is a little light here.

Comment (loic) [inline docs/ref/forms/validation.txt]: It'll also work in `Model.clean()` obviously but these are the forms docs, which is a bit of a problem, `Model.clean()` is actually the only place where this pattern is recommended, `Form.clean()` should use `Form.add_error()` instead.

Comment (timgraham) [inline docs/ref/forms/validation.txt]: This new snippet should be moved to the model validation docs, then, correct?

**Comment by timgraham** [comment]
Updated and merged in 52a190b65781f8dc07abd230aaf9043fbdbf4fba, thanks!

**Pull request closed without merging**